duckduckgo/duckduckgo-privacy-extension
# Further copy changes to popup and options page

**Reviewer:** @alohaas 



## Description:
In the main popup:
Manage protection ---> Unprotected sites

In the options page for the empty state:
Privacy Protection enabled for all sites ---> No Unprotected Sites Yet


## Steps to test this PR:
1. Build and load extension
2. Verify copy changes as per description

## Automated tests:

###### Reviewer Checklist:


###### PR Author Checklist:
